Checkout updates blog post (#1660)
carpentries/carpentries.org: _posts/2023/08/2023-08-14-checkout_changes.md

via email or Google form. 
We will continue to encourage new trainees to engage with lesson maintenance and improvement! 
However, for those with other interests, there are now many more ways to "get involved" with The Carpentries for this step.

**Previously: Trainees were required to make a contribution to the content or conversation around our lessons and other resources on GitHub.** 
This was intended to support Instructors in learning to share feedback that contributes to the maintenance of our materials. 
However, many trainees found this step to be a barrier to completing checkout, and we could not find clear evidence that that this step was achieving its intended goals.  

**Now: Lesson contributions will no longer be required, but will continue to be welcomed as part of the checkout process.** 
The "Get Involved" step allows Trainees to choose from a variety of options, with the goal that they will take the next step in their involvement in The Carpentries community. 
Any participation will be accepted, but we suggest one of the following activities:
- [ ] Serve as an Instructor or a Helper at a Carpentries workshop

> - [ ] Attend an Instructor Meeting, regional call, or other community event

Author: ```suggestion
- [ ] Attend a regional call, skill-up session, or other community event
```

- [ ] Submit a contribution to a Lesson, Glossario, or other Carpentries repository

Signup procedures will vary by activity. Once an activity is completed, it must be submitted for credit directly in [AMY, our database](https://amy.carpentries.org) (discussed below).

### Tracking Checkout progress

> **Previously: Teaching Demonstrations and Community Discussions were automatically recorded. After submitting a lesson contribution, trainees could receive credit by reporting their contribution in AMY, our database, or by emailing it to the Instructor Training team.**  

Author: ```suggestion
Previously: **Most types of Community Sessions were automatically recorded as meeting the Community Discussion requirement. After submitting a lesson contribution, trainees could receive credit by sharing a link in AMY (our database) or by emailing it to the Instructor Training team.**  
```

> **Now: Teaching Demonstrations and Welcome Sessions will be automatically recorded for each trainee. After completing a "Get Involved" activity, trainees must [login to AMY](https://amy.carpentries.org) to report which participation option they wish to have credited.**  [Read more about how to access and update your AMY profile](https://carpentries.github.io/amy/users_guide/community_index/)

Author: ```suggestion
**Now: Teaching Demonstrations and Welcome Sessions will continue to be automatically recorded for each trainee. After completing a "Get Involved" activity, trainees must [login to AMY](https://amy.carpentries.org) to report which participation option they wish to have credited.**  [Read more about how to access and update your AMY profile](https://carpentries.github.io/amy/users_guide/community_index/)
```

| Community Discussion | Sign up for a [Welcome Session](https://pad.carpentries.org/welcome-sessions-2023) (or refer to FAQ below)       | Counts as Welcome Session, or next FAQ|
| Lesson contribution    | Review recommended Carpentries activities for your "Get Involved" step                                                                      | Counts as "Get Involved" step |

### I am in the process of completing checkout and have already finished my Community Discussion requirement. How will that be applied?
If you have already attended a Community Discussion (or other commumity event) as part of checkout, we have automatically transferred this activity to complete your Welcome Session step.

This means:
- You do not need to attend a Welcome Session unless you want to.
- If you do attend a Welcome Session, we will use the Welcome Session towards your Welcome Session requirement and the previously attended community Discussion to complete your Get Involved step.

### I am in the process of completing checkout. I haven't completed my Community Discussion but I had planned to attend a Community Session rather than Welcome Session. Do I have to attend a Welcome Session?

> **Community Session attendance will be accepted as a means to fulfill the Welcome Session step through 31 October, 2023.** However, from this point forward, Community Discussion attendance will **not** automatically be counted towards this step because some trainees will choose to submit these for their "Get Involved" step instead. If you would like to have a Community Discussion that is not a Welcome Session counted for your Welcome Session step, please contact us at instructor.training@carpentries.org.

Author: ```suggestion
**Community Session attendance will be accepted as a means to fulfill the Welcome Session step through 14 November, 2023.** However, from this point forward, Community Discussion attendance will **not** automatically be counted towards this step because some trainees will choose to submit these for their "Get Involved" step instead. If you would like to have a Community Discussion that is not a Welcome Session counted for your Welcome Session step, please contact us at instructor.training@carpentries.org.
```

### I am experiencing a barrier to accessibility during the Instructor checkout process. How can I get help?
We want to help! Please use our [accommodation request form](https://carpentries.typeform.com/to/B2OSYaD0) and a member of our team will get back to you. 

### I was trained a long time ago and never completed checkout. These new rules seem more achievable to me. Can I re-start my checkout process?
These changes are being made, in part, because we recognize that the challenges of our checkout process have prevented skilled and enthusiastic Instructors from completing their certification. 
However, it is also important that trainees begin their Instructor role with Carpentries' training and ethos fresh in their mind. 

> Now through **31 October, 2023**, we will offer the following accommodations for anyone previously trained:

Author: ```suggestion
Now through **14 November, 2023**, we will offer the following accommodations for anyone previously trained:
```
